### DISCUSSION TITLE
1.2.0-alpha.1

### DISCUSSION THREAD

**Discussion by jasonsaayman** [axios/axios#5248] [Announcements]
Hi All

Please can you try the pre-release by running:

```
npm i axios@1.2.0-alpha.1
```

Please leave your feedback in this discussion.

**Comment by agantelin** [comment]
The nasty bug with headers seems to be gone. Thank you!

**Comment by LegendSebastiano-L** [comment]
the issue in https://github.com/axios/axios/issues/5243 is still present
